src/components/migrationToV3.stories.mdx

```diff
@@ -19,20 +19,25 @@ npm install --save amazon-chime-sdk-component-library-react@beta amazon-chime-sd
 
 ## What's New
 
-- `MeetingManager`'s `join()` method no longer takes an `EventReporter` and instead takes an `EventController`.
-- Add `MeetingSessionConfiguration` as a required parameter to `MeetingManager.join()` method. With this change the builders have direct access to `MeetingSessionConfiguration`, this will allow more flexibility to customize the `MeetingSession`.
-- Add `MeetingManagerJoinOptions` as a new interface for the `options` parameter of the `MeetingManager.join` method.
+- Add a new interface `MeetingManagerJoinOptions` containing `deviceLabels`, `eventController`, `enableWebAudio`, and `activeSpeakerPolicy` properties. This interface is used for optional parameter `options` of `MeetingManager.join()` API.
```

Comment:
I realized now, this list is extensive, do you think it would be good to have sections of Added, Changed and Removed in here? Or may be just link to changelog for this release? I think this is redundant with the changelog right?

Reply:
Yeah, I think we just copy CHANGELOG items to here. I gonna remove these bullet points and use the link to `CHANGELOG` instead.

Reply:
Copied the summarize of V3 Changes to here and added a link to CHANGELOG.